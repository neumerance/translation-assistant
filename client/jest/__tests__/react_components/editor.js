import React from 'react';

import OtgsWebtaEditor from '../../../app/bundles/App/components/cms/partials/text_editor/otgs_webta_editor/editor';
import renderer from 'react-test-renderer';

it('renders', () => {
  let originalString = 'hello string';
  let editorController = {
    outerDom: {
      editorCssId: 'test-css-id'
    },
    prepare: ()=> {}
  };
  let dbid = 1;
  let colorSelectionData = {};
  let xtagTokenData = {};

  const component = renderer.create(
    <OtgsWebtaEditor originalString={ originalString }
                     controller={ editorController }
                     dbid={ dbid }
                     colorSelectionData={ colorSelectionData }
                     xtagTokenData={ xtagTokenData }
    />
  );

  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});

