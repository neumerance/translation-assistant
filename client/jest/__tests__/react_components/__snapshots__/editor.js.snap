// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="otgs-webta-editor"
  dangerouslySetInnerHTML={
    Object {
      "__html": "hello string",
    }
  }
  id="test-css-id"
/>
`;
